pygis/docs/e_vector_overlay.md

```diff
@@ -0,0 +1,381 @@
+---
+jupytext:
```

Comment:
Same

Reply:
Done--all feedback from other pull request resolved! Changes for e_vector_overlay.md committed/pushed.

Reply:
Standing by for updated graphics.